common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -122,9 +142,12 @@ public boolean containsValue(Object value) {
      *
      * @param <V> the value type stored in the map.
      */
-    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V>,
-            Iterable<IntObjectMap.Entry<V>> {
+    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V> {
         final IntObjectMap<V> map;
+        Set<Integer> keySet;
```

Comment:
The point I am trying to focus on is just because this is an inner class doesn't mean we should use different rules for the internals (unless there is a reason to... i.e. avoid synthetic method generation).

[DefaultHttp2Connection.java#L839](https://github.com/netty/netty/blob/master/codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java#L839).  The access level does not have to be package private to access them outside the scope, but this is typically done if the member is used outside the scope of the inner class to avoid the potential of synthetic member generation to access an inner private member of the class.

Here is another example:
```java
public class AccessLevelTest {
    private AccessLevelTest() {
        InnerClass inner = new InnerClass();
        int x = inner.a + inner.b + inner.c + inner.d;
        System.out.println("x=" + x);
    }

    private static final class InnerClass {
        private int a = 1;
        int b = 2;
        protected int c = 3;
        public int d = 4;
    }

    public static void main(String[] args) {
        new AccessLevelTest();
    }
}
```